Sming/appinit/user_main.cpp

```diff
@@ -12,7 +12,7 @@ extern void init();
 extern "C" void  __attribute__((weak)) user_init(void)
 {
 	system_timer_reinit();
-	uart_div_modify(UART_ID_0, UART_CLK_FREQ / 115200);
+
```

Comment:
@ADiea please, remove the newline between system_timer_reinit and cpp_core_initialize. After that rebase on develop and ping me to merge the PR.